assets/tests/components/routeLadder.test.tsx

```diff
@@ -225,6 +225,39 @@ describe("routeLadder", () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test("renders a route ladder with the new header and detour dropdown", () => {
+    jest
+      .mocked(getTestGroups)
+      .mockReturnValue(["route-ladder-header-update", "detours-pilot"])
```

Comment:
Could you import the `TestGroups` enum and reference them via the symbols instead?
```suggestion
      .mockReturnValue([TestGroups.RouteLadderHeaderUpdate, TestGroups.DetoursPilot])
```